src/plugins/internalnotification/README.md

```diff
@@ -0,0 +1,37 @@
+- infos = Plugin for internal notification
+- infos/author = Thomas Wahringer <waht@libelektra.org>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides =
+- infos/recommends =
+- infos/placements = postgetstorage postcommit
+- infos/status = global libc unittest concept unfinished experimental nodoc
+- infos/metadata =
+- infos/description = Plugin for internal notification
+
+## Usage ##
+
+Allows programs to register integer variables that get automatically updated
+when the value of the key changes.
```

Comment:
That would imply that the integer gets also updated when any other key (and not the one for the integer) in the database changes. I haven't read through everything yet, but I doubt this is the case.